schema/bom-1.6.schema.json

```diff
@@ -661,34 +661,35 @@
             {
               "type": "array",
               "title": "Creation Tools (legacy)",
-              "description": "[Deprecated] The tool(s) used in the creation of the BOM.",
+              "description": "[Deprecated] The tool(s) used in the creation of the CycloneDX document.",
               "items": {"$ref": "#/definitions/tool"}
             }
           ]
         },
         "authors" :{
           "type": "array",
           "title": "Authors",
-          "description": "The person(s) who created the BOM. Authors are common in BOMs created through manual processes. BOMs created through automated means may not have authors.",
+          "description": "The person(s) who created the CycloneDX document.\nAuthors are common in documents created through manual processes. Documents created through automated means may not have authors.\nThis may be different from the author(s) of the component that the CycloneDX document describes.",
           "items": {"$ref": "#/definitions/organizationalContact"}
         },
         "component": {
           "title": "Component",
-          "description": "The component that the BOM describes.",
+          "description": "The component that the CycloneDX document describes.",
           "$ref": "#/definitions/component"
         },
         "manufacture": {
-          "title": "Manufacture",
-          "description": "The organization that manufactured the component that the BOM describes.",
+          "title": "Manufacturer",
+          "description": "The organization that manufactured the CycloneDX document (the \"manufacturer\", although the property is misspelled).\nThis may be different from the manufacturer of the component that the CycloneDX document describes.",
           "$ref": "#/definitions/organizationalEntity"
         },
         "supplier": {
           "title": "Supplier",
-          "description": " The organization that supplied the component that the BOM describes. The supplier may often be the manufacturer, but may also be a distributor or repackager.",
+          "description": " The organization that supplied the CycloneDX document. The supplier may often be the manufacturer, but may also be a distributor or repackager.\nThis may be different from the supplier of the component that the CycloneDX document describes.",
```

Comment:
:exclamation: this is a breaking change in semantic of the spec.

Reply:
Agreed and noted.